test/format-test.cc

```diff
@@ -1006,12 +1007,12 @@ TEST(format_test, precision) {
   EXPECT_THAT(outputs,
               testing::Contains(fmt::format("{:.838A}", -2.14001164E+38)));
 
-  if (std::numeric_limits<long double>::digits == 64) {
-    auto ld = (std::numeric_limits<long double>::min)();
-    EXPECT_EQ(fmt::format("{:.0}", ld), "3e-4932");
-    EXPECT_EQ(fmt::format("{:0g}", 5.02957084971264961283E-4940L),
-              "5.02957e-4940");
-  }
+#if LDBL_MANT_DIG == 64
```

Comment:
I suggest keeping the normal if statement but replacing `5.02957084971264961283E-4940L` with `std::numeric_limits<long double>::denorm_min()` because the exact denormal value doesn't matter for this test.

Reply:
Done